docs/zms_client.md

```diff
@@ -220,7 +223,7 @@ following zms-cli command:
 
     $ zms-cli -d <domain> add-group-role <role> <member> [<member> ...]
 
-### Parameters
+<h3 id="parameters-1">Parameters</h3>
```

Comment:
Links didn't work for Parameters and examples as the id wasn't recognized. Solved it by using HTML headers (h3 is equivalent to ###)